### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by jc740631**
\@Raamthapa1
Please check formatting, naming and logic

### CODE REVIEW

**Review by Raamthapa1 — APPROVED** [review]
Please Merge your branch to master

File: week 5/emails.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def get_name(email):
+    prefix = email.split('@')[0]
+    parts = prefix.split('.')
+    name = " ".join(parts).title()
+    return name
+
+def main():
```
Comment (lindsaymarkward) [inline week 5/emails.py:7]: main() ALWAYS goes first.

File: week 5/emails.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    email_detail = {}
```
Comment (lindsaymarkward) [inline week 5/emails.py:8]: email_detail is not how we name dictionaries

File: week 5/state_names.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "TAS": "Tasmania"
+}
+
+print(CODE_TO_NAME)
+print(CODE_TO_NAME["QLD"])
+print(CODE_TO_NAME["VIC"])
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
```
Comment (lindsaymarkward) [inline week 5/state_names.py:24]: You needed to use exceptions

**Pull request merged**